docs/other-topics/legacy.md

```diff
@@ -54,6 +54,24 @@ Collection.init({
 
 And if your model has no primary key at all you can use `Model.removeAttribute('id');`
 
+Instances without primary keys can still be retrieved using `Model.findOne` and `Model.findAll`.  
+While it's currently possible to use their instance methods (`instance.save`, `instance.update`, etc…), doing this will lead to subtle bugs, 
+and is planned for removal in a future update.
```

Comment:
note: 'future update.' is 'future major update.' in the v6 docs because I'd like to remove this in v7

Reply:
The text about what the subtle bugs are, which I proposed, is gone.